index.html

```diff
@@ -20,24 +20,25 @@
         </li>
       </ul>
     </div>
```

Comment:
Kindly use `nav` tag inside `header` tag for your navbar section, that will make your code more semantic.